frontend/src/views/CellCards/components/CellCard/components/OntologyDagView/components/AnimatedNodes/index.tsx

```diff
@@ -0,0 +1,149 @@
+      n_cells_rollup: number;
+    };
+  }) => void;
+  hideTooltip: () => void;
+}
+export default function AnimatedNodes({
+  tree,
+  cellTypeId,
+  duration,
+  setDuration,
+  toggleTriggerRender,
+  showTooltip,
+  hideTooltip,
+}: AnimatedNodesProps) {
+  const cellTypesById = useCellTypesById() || {};
+  const handleMouseOver = (
+    event: React.MouseEvent<SVGElement>,
+    datum: TreeNodeWithState
+  ) => {
+    if (event.target instanceof SVGElement) {
+      if (event.target.ownerSVGElement !== null) {
+        const coords = localPoint(event.target.ownerSVGElement, event);
+        if (coords) {
+          showTooltip({
+            tooltipLeft: coords.x,
+            tooltipTop: coords.y,
+            tooltipData: {
+              n_cells: datum.n_cells,
+              n_cells_rollup: datum.n_cells_rollup,
+            },
+          });
+        }
+      }
+    }
+  };
+
+  return (
+    <NodeGroup
+      data={tree.descendants()}
+      keyAccessor={(d: HierarchyPointNode<TreeNodeWithState>) => d.data.id}
+      start={(node: HierarchyPointNode<TreeNodeWithState>) => {
+        return node.parent
+          ? {
+              top: node.parent.data.x0 ?? node.parent.x,
+              left: node.parent.data.y0 ?? node.parent.y,
+              opacity: 0,
+              timing: { duration },
+            }
+          : {
+              top: node.x,
+              left: node.y,
+              opacity: 0,
+              timing: { duration },
+            };
+      }}
+      enter={(node: HierarchyPointNode<TreeNodeWithState>) => {
+        return {
+          top: [node.x],
+          left: [node.y],
+          opacity: [1],
+          timing: { duration },
+        };
+      }}
+      update={(node: HierarchyPointNode<TreeNodeWithState>) => {
+        return {
+          top: [node.x],
+          left: [node.y],
+          opacity: [1],
+          timing: { duration },
+        };
+      }}
+      leave={() => {
+        return { opacity: [0], timing: { duration: 0 } };
+      }}
+    >
+      {(nodes) => {
+        return (
+          <Group>
+            {nodes.map(({ key, data: node, state }) => {
+              const isInCorpus =
+                (node.data.id.split("__").at(0)?.replace("_", ":") ?? "") in
+                cellTypesById;
+              return (
+                <Node
+                  key={`${key}-node`}
+                  isInCorpus={isInCorpus}
+                  animationKey={key}
+                  node={node}
+                  isTargetNode={node.data.id.split("__").at(0) === cellTypeId}
+                  handleMouseOver={handleMouseOver}
+                  handleMouseOut={hideTooltip}
+                  maxWidth={NODE_SPACINGS[1] - 20}
+                  left={state.left}
+                  top={state.top}
+                  opacity={state.opacity}
+                  handleClick={() => {
+                    if (duration === 0) {
+                      setDuration(TREE_ANIMATION_DURATION);
```

Comment:
after the first load, set the duration to "TREE_ANIMATION_DURATION". this is so that we don't see any animation when the component first loads